src/components/shared/card.tsx

```diff
@@ -150,10 +152,67 @@ const cardStyles = (itemType: RelatedItemType, format: Format): SerializedStyles
     }
 }
 
-const Card = ({ relatedItem, image }: Props): JSX.Element => {
+const parentIconStyles: SerializedStyles = css`
+    display:inline-block;
+    svg {
+        width: 0.875rem;
+        height: auto;
+        margin-left: auto;
+        margin-right: auto;
+        margin-top: 0.300rem;
+        display: block;
+    }
+`;
+
+const iconStyles = (format: Format): SerializedStyles => {
+    const { inverted } = getPillarStyles(format.pillar);
+    return css`
+        width: 1.5rem;
+        height: 1.4375rem;
+        display: inline-block;
+        background-color: ${inverted};
+        border-radius: 50%;
+    `;
+}
 
+const icon = (itemType: RelatedItemType, format: Format): JSX.Element => {
+    if (itemType === RelatedItemType.GALLERY) {
```

Comment:
Could this be a switch statement rather than an if/else? To make it easier to read